WordPress/Classes/ViewRelated/Stats/Shared Views/Stats Detail/SiteStatsInsightsDetailsViewModel.swift

```diff
@@ -527,8 +529,8 @@ class SiteStatsInsightsDetailsViewModel: Observable {
                 forceRefresh: forceRefresh))
     }
 
-    func refreshFollowers() {
-        ActionDispatcher.dispatch(InsightAction.refreshFollowers)
+    func refreshFollowers(forceRefresh: Bool = true) {
+        ActionDispatcher.dispatch(InsightAction.refreshInsights(forceRefresh: forceRefresh))
```

Comment:
Do I understand correctly that all the possible insight data is refreshed when a user opens "Total Followers" view?

Reply:
That's correct